client/login/qr-code-login/style.scss

```diff
@@ -0,0 +1,48 @@
+@import '@automattic/onboarding/styles/mixins';
+@import '@wordpress/base-styles/mixins';
+
+.qr-code-login {
+    margin: 0 auto;
+	max-width: 400px;
+    text-align: center;
+}
+
+.qr-code-login__heading {
+    font-size: $font-title-small;
+	margin-bottom: 32px;
+	margin-top: 16px;
+	text-align: center;
+}
+
+.qr-code-login__card canvas {
+    width: 100%;
+    height: auto;
+}
+
+.qr-code-login__footer {
+    a {
+		border-bottom: 1px solid #c8d7e1;
```

Comment:
Could we use an existing variable for this color? Otherwise those are impossible to update together with a color scheme update.

Reply:
Since there is only one link right now I removed this border.